components/script/dom/htmlimageelement.rs

```diff
@@ -1562,7 +1626,14 @@ impl HTMLImageElementMethods for HTMLImageElement {
         let request = self.current_request.borrow();
         let request_state = request.state;
         match request_state {
-            State::CompletelyAvailable | State::Broken => true,
+            State::CompletelyAvailable => {
```

Comment:
Same here, this is not the point where the request becomes broken or completely available.